### PR TITLE
Adds Dockerfile for citest dependencies, additional DC/OS citests

### PR DESCRIPTION
**Pull request by mtweten**

### CODE REVIEW

**Review by TrevinTeacutter — APPROVED** [review]

File: testing/citest/tests/dcos_smoke_test.py [hunk 1]
```diff
@@ -178,31 +150,108 @@ def delete_load_balancer(self):
...
+            'cloudProvider': 'dcos',
+            'application': self.TEST_APP,
+            'account': bindings['SPINNAKER_DCOS_ACCOUNT'],
+            'env': {},
+            'desiredCapacity': 1,
+            'cpus': 0.1,
+            'mem': 64,
+            'docker': {
+              'image': {
+                'repository': 'wg9059/nginx-test',
```
Comment (willgorman) [inline testing/citest/tests/dcos_smoke_test.py]: Should we use a publicly available image for this?

File: testing/citest/tests/dcos_smoke_test.py [hunk 2]
```diff
@@ -178,31 +150,108 @@ def delete_load_balancer(self):
...
+    return st.OperationContract(
+        self.new_post_operation(
+            title='create_server_group', data=payload, path='tasks'),
+        contract=builder.build())
+
+
+  def delete_server_group(self, version='v000'):
+    """Creates OperationContract for deleteServerGroup.
+
+    To verify the operation, we just check that the Kubernetes container
```
Comment (willgorman) [inline testing/citest/tests/dcos_smoke_test.py]: DC/OS container?

File: testing/citest/tests/dcos_smoke_test.py (hunk 1 above)
Comment (mtweten) [reply testing/citest/tests/dcos_smoke_test.py]: Switched to nginx from docker.io

File: testing/citest/tests/dcos_smoke_test.py (hunk 2 above)
Comment (mtweten) [reply testing/citest/tests/dcos_smoke_test.py]: done

**Pull request merged**